hoomd/hpmc/update.py

```diff
@@ -14,388 +14,171 @@
-        self.setupUpdater(period);
-
-        self.volume_delta = 0.0;
-        self.volume_weight = 0.0;
-        self.ln_volume_delta = 0.0;
-        self.ln_volume_weight = 0.0;
-        self.length_delta = [0.0, 0.0, 0.0];
-        self.length_weight = 0.0;
-        self.shear_delta = [0.0, 0.0, 0.0];
-        self.shear_weight = 0.0;
-        self.shear_reduce = 0.0;
-        self.aspect_delta = 0.0;
-        self.aspect_weight = 0.0;
-
-        self.metadata_fields = ['betaP',
-                                 'seed',
-                                 'volume_delta',
-                                 'volume_weight',
-                                 'ln_volume_delta',
-                                 'ln_volume_weight',
-                                 'length_delta',
-                                 'length_weight',
-                                 'shear_delta',
-                                 'shear_weight',
-                                 'shear_reduce',
-                                 'aspect_delta',
-                                 'aspect_weight']
-
-    def set_betap(self, betaP):
-        R""" Update the pressure set point for Metropolis Monte Carlo volume updates.
-
-        Args:
-            betaP (float) or (:py:mod:`hoomd.variant`): :math:`\frac{p}{k_{\mathrm{B}}T}`. (units of inverse area in 2D or
-                inverse volume in 3D) Apply your chosen reduced pressure convention
-                externally.
-        """
-        self.betaP = hoomd.variant._setup_variant_input(betaP)
-        self.cpp_updater.setP(self.betaP.cpp_variant)
-
-    def volume(self, delta=None, weight=None):
-        R""" Enable/disable isobaric volume move and set parameters.
-
-        Args:
-            delta (float): maximum change of the box area (2D) or volume (3D).
-            weight (float): relative weight of this box move type relative to other box move types. 0 disables this move type.
-
-        Sample the isobaric distribution of box volumes by rescaling the box.
-
-        Note:
-            When an argument is None, the value is left unchanged from its current state.
-
-        Example::
-
-            box_update.volume(delta=0.01)
-            box_update.volume(delta=0.01, weight=2)
-            box_update.volume(delta=0.01, weight=0.15)
-
-        Returns:
-            A :py:class:`dict` with the current values of *delta* and *weight*.
-
-        """
-        self.check_initialization();
-
-        if weight is not None:
-            self.volume_weight = float(weight)
-
-        if delta is not None:
-            self.volume_delta = float(delta)
-
-        self.cpp_updater.volume(self.volume_delta, self.volume_weight);
-        return {'delta': self.volume_delta, 'weight': self.volume_weight};
+            * ``weight`` (float) - relative weight of log(V) box moves relative to other box move types.
+            * ``delta`` (float) - maximum change of **ln(V)** (where V is box area (2D) or volume (3D)).
```

Comment:
It's weird that one line says "log" and the other says "ln." This should be made consistent.